File: February 14, 2026.py
def get_difficulty(track):

    points = 0
    lastTurn = None

    for turn in track:
        if turn in ("L", "R"):
            if lastTurn in ("L", "R"):
                if lastTurn != turn:
                    points += 15
                else:
                    points += 5
            else:
                points += 5
            lastTurn = turn
        else:
            lastTurn = None

    if points >= 200:
        return "Hard"
    elif points > 100:
        return "Medium"
            
    return "Easy"

File: February 14, 2026.js
function getDifficulty(track) {

  let points = 0;
  let lastTurn = null;

  for (let turn of track) {
    
    if(turn == "L" || turn == "R") {
      if(lastTurn == "L" || lastTurn == "R") {
          if(lastTurn != turn) {
            points += 15;
          } else {
            points += 5;
          }
        } else {
          points += 5;
        }

        lastTurn = turn;

      } else {
        lastTurn = null;
      }

    }

  if (points >= 200) {
    return "Hard"
  } else if (points > 100) {
    return "Medium"
  }

  return "Easy";
}